packages/salesforcedx-apex-debugger/test/integration/adapter.test.ts

```diff
@@ -201,7 +201,7 @@ const execApexNoWait = (
   userName: string
 ): CommandExecution => {
   return new CliCommandExecutor(
-    new SfdxCommandBuilder()
+    new SfCommandBuilder()
       .withArg('force:apex:execute')
```

Comment:
Can this be changed to `apex:run`?  If yes, can you also change the flags to SF-style?